### DISCUSSION TITLE
关于每次启动窗口大小的重置

### DISCUSSION THREAD

**Discussion by juzizhen** [MCSLTeam/MCSL2#126] [功能建议] [answered]
<h3>我建议在关闭窗口时自动记录窗口大小，并在下次启动时自动调整窗口大小到先前记录的大小。</h3>
这真的很重要，每次启动都要重新调整大小😢

**Comment by lxhtt** [comment, marked as answer]
add in https://github.com/MCSLTeam/MCSL2/commit/739954abfcde1077819c35c8e1fcf81b3dd5fd91
